src/utils/csv.ts

```diff
@@ -0,0 +1,9 @@
+import * as fs from "fs";
```

Comment:
We use explicit named imports/destructured rather than importing `*`
```suggestion
import { createReadStream } from "fs";
```

Reply:
fix in [71fbe64](https://github.com/sparkletown/sparkle/pull/1613/commits/71fbe64d1f3591d78979411e27e72e5165340a4b)